local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,261 @@
+                    await _stop_current_job()
+
+        finally:
+            await _stop_current_job()
+            sender_task.cancel()
+
+
+async def main():
+    MODEL_PATH = "/home/amadeus/models/qwen3_tts"
+    # MODEL_PATH = "/mnt/h/pr/N.E.K.O/local_server/qwen3_tts_server/Qwen3-TTS/pretrained_model/Qwen3-TTS-12Hz-1.7B-Base"
+    server_instance = QwenLocalServer(MODEL_PATH)
+    async with websockets.serve(server_instance.handle_tts, "0.0.0.0", 8765):
```

Ruff (0.14.14)

[error] Possible binding to all interfaces

(S104)